frontend/src/components/user/UserControl.tsx

```diff
@@ -132,8 +134,11 @@ export const UserControl = () => {
     });
 
     logoutLocalUser();
+
+    setQueryParam("auth_logout", "ok");
```

Comment:
Can we change this to follow the pattern of the rest of the app and use camelCase for query param keys? 

Also, since this param is used by the app to determine when to redirect a user upon logout, a more descriptive name, such as `redirectOnLogout` would be nice

Lastly, I think "true" is more in line with general query param value practices, can we use that instead of "ok"?